docs/website/docs/general-usage/incremental-loading.md

```diff
@@ -348,6 +348,20 @@ pipeline.run(dim_customer())  # second run — 2024-04-09 22:13:07.943703
 | 2024-04-09 18:27:53.734235 | NULL | 2 | bar | 2 |
 | **2024-04-09 22:13:07.943703** | **NULL** | **1** | **foo_updated** | **1** |
 
+:::tip
+To disable the prevention of retiring absent records,
```

Comment:
Do you mean if they decide to roll it back? So the scenario would be:
* user specifies `merge_key` to prevent retirement of absent rows
* user changes their mind, and decides to remove this hint from the resource decorator for the same pipeline
?

If so, then I suggest we explicitly say: 
```
If you decide to revert this change for an existing pipeline and start retiring absent records, ...
```